### PR TITLE
OpenStack GPU support

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Document how GPUs should be configured to be available in OpeStack.

---

**Related issue :**

### CODE REVIEW

File: content/en/providers/cloud-compute/openstack/gpu.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+This allows to plug any kind of PCI device to the Virtual Machines.
+
+As a summary of the OpenStack documentation, these are the steps needed to add a
+GPU enabled flavor (be aware this may need tuning to your specific
+hardware/configuration!):
+
+1. On computing node, get vendor/product ID of your hardware:
+   `lspci | grep NVDIA` to get pci slot of GPU, then
+   `virsh nodedev-dumpxml pci_xxxx_xx_xx_x`
+1. On computing node, unbind device from host kernel driver
```
[the remark this answers is not in the document]
Comment (enolfc) [reply content/en/providers/cloud-compute/openstack/gpu.md:22]: I'm clueless about the specifics. So I would just bring any suggestions you may have :)

[the remark this answers is not in the document]
Comment (tdviet) [reply content/en/providers/cloud-compute/openstack/gpu.md:22]: Sorry for this later reply. Unbinding is system dependent, and can be done in many ways:
- if the kernel does not uses the devices (no GPU drivers included in kernel, or drivers disable in GRUB), nothing to unbind
- via pci-stub `grubby --args="pci-stub.ids=10de:11fa" --update-kernel DEFAULT` like in the RedHat manual, section 12.1, step 1-2; where the `pci-stub.ids` value is `vendor_ID: product_id` from `lspci`.
- via echo command `echo $dev > /sys/bus/pci/devices/$dev/driver/unbind` where `$dev` is the PCI device ID `xx:xx.x`  or `xxxx:xx:xx.x` from `lspci`

[the remark this answers is not in the document]
Comment (enolfc) [reply content/en/providers/cloud-compute/openstack/gpu.md:22]: ok, added this.

**Comment by enolfc** [comment]
@gwarf, is it still ok to merge?

**Pull request merged**